perfkitbenchmarker/benchmarks/hbase_ycsb_benchmark.py

```diff
@@ -93,14 +93,16 @@ def CheckPrerequisites():
   ycsb.CheckPrerequisites()
 
 
-def _CreateYCSBTable(vm, table_name=TABLE_NAME, family=COLUMN_FAMILY,
-                     n_splits=TABLE_SPLIT_COUNT):
+def CreateYCSBTable(vm, table_name=TABLE_NAME, family=COLUMN_FAMILY,
```

Comment:
If this is being used across benchmarks, should it be pulled into one of the packages (hbase.py or ycsb.py)?